packages/headless/src/api/commerce/recommendation/recommendation-request.ts

```diff
@@ -0,0 +1,52 @@
+import {BaseParam} from '../../platform-service-params';
+import {
+  ClientIdParam,
+  ContextParam,
+  CountryParam,
+  CurrencyParam,
+  LanguageParam,
+  PageParam,
+  QueryParam,
+  SlotIdParam,
+  TrackingIdParam,
+} from '../commerce-api-params';
+import {CommerceAPIRequest, baseRequest} from '../common/request';
+
+export type CommerceSearchRequest = CommerceAPIRequest & QueryParam;
+
+export type CommerceRecommendationRequest = BaseParam &
+  SlotIdParam &
+  TrackingIdParam &
+  LanguageParam &
+  CountryParam &
+  CurrencyParam &
+  ClientIdParam &
+  ContextParam &
+  PageParam;
```

Comment:
These parameters are all common to commerce requests. Can we extract them to an interface that can be used by other commerce requests? 👀